Brewfile

```diff
@@ -1 +1,3 @@
 brew 'Swiftlint'
+tap "danger/tap"
+brew "danger-swift"
```

Comment:
are these changes still needed, given Danger is provided by the `417-72KI/danger-swiftlint@v3` action?

Reply:
@p4checo do you think we should keep Swiflint? It is now integrated with Danger, maybe we could remove it too.

Reply:
I think we can. 

It was added here during the initial SwiftLint integration in the project, but funnily enough it was never invoked via CI 😅

We also run SwiftLint locally in the project, which is done via a SwiftLint target that contains a shell script to run `swiftlint` and print a warning if it isn't installed. In hindsight, this script should trigger an error instead of just printing... 🤔